src/utils/pipes/parse.array.pipe.ts

```diff
@@ -11,7 +11,7 @@ export class ParseArrayPipe implements PipeTransform<string | undefined, Promise
   transform(value: string | undefined, _: ArgumentMetadata): Promise<string[] | undefined> {
     return new Promise(resolve => {
       if (value === undefined || value === '') {
-        return resolve(undefined);
+        return resolve([""]);
```

Comment:
this may cause unintended consequences in endpoints that accept arrays. There are checks for undefined where the filters are applied which might break the existing functionality